code/modules/mob/living/brain/brain_item.dm

```diff
@@ -113,24 +113,31 @@
 	if(istype(O, /obj/item/borg/apparatus/organ_storage))
 		return //Borg organ bags shouldn't be killing brains
 
-	if((organ_flags & ORGAN_FAILING) && O.is_drainable() && O.reagents.has_reagent(/datum/reagent/medicine/mannitol)) //attempt to heal the brain
+	var/healby = 2 //heals 2 damage per unit of mannitol
+	var/pour = "carefully pour"
+	var/onto = "onto"
+	if(istype(O, /obj/item/reagent_containers/syringe))
+		healby = 4 //heal 4 damage because PRECISCION
```

Comment:
true, but you're also jabbing their brain with a needle, no?

Reply:
It's a PRECISE needle, ATH